src/main/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/incidents/IncidentResource.kt

```diff
@@ -128,6 +132,67 @@ class IncidentResource(private val incidentService: IncidentService) {
   fun getIncident(
     @Schema(description = "Incident id") @PathVariable incidentId: Long,
   ) = incidentService.getIncident(incidentId)
+
```

Comment:
I will also be getting a list of open incident ids per prison (to be able to then compare open incident details) but this will be a separate PR